From 5351875cde23340c0a98fe46566cde5535c40e21 Mon Sep 17 00:00:00 2001
From: georgev93 <georgeavogt93@gmail.com>
Date: Fri, 26 Jun 2020 19:07:14 -0400
Subject: [PATCH 1/2] Call cmake_minimum_required() before project() in
 CMakeLists.txt.

From https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html:
Note: Call the cmake_minimum_required() command at the beginning of the top-level CMakeLists.txt file even before calling the project() command. It is important to establish version and policy settings before invoking other commands whose behavior they may affect. See also policy CMP0000.

Signed-off-by: Moritz Haase <Moritz.Haase@bmw.de>
Upstream-Status: Backport [e0c050d8b614294b2215a62e64628470fab67f20]
---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 88e0cfb5..d36c6491 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,12 +1,12 @@
+# 2.6.3 is needed for ctest support
+# 3.1 is needed for target_sources
+cmake_minimum_required(VERSION 3.1)
+
 project(CppUTest)
 
 set(CppUTest_version_major 4)
 set(CppUTest_version_minor 0)
 
-# 2.6.3 is needed for ctest support
-# 3.1 is needed for target_sources
-cmake_minimum_required(VERSION 3.1)
-
 ###############
 # Conan support
 ###############
